ostelco-ios-client/Coordinators/OnboardingCoordinator.swift

```diff
@@ -188,7 +188,8 @@ class OnboardingCoordinator {
 }
 
 extension OnboardingCoordinator: LoginDelegate {
-    func signedIn(authCode: String, contactEmail: String?) {
+    func signedIn(controller: UIViewController, authCode: String, contactEmail: String?) {
+        controller.showSpinner()
```

Comment:
You'll need to remove the spinner in the case where sign in fails and we can't go to the next screen.

Reply:
Added `controller.removeSpinner(spinnerView)` in `catch`